src/tests/ftest/control/config_generate_run.py

```diff
@@ -64,6 +65,11 @@ def test_config_generate_run(self):
         except yaml.YAMLError as error:
             self.fail(f"Error loading dmg generated config! {error}")
 
+        # Iterate & update the log file path for each engine.
+        engines = generated_yaml["engines"]
+        for engine in engines:
+            engine["log_file"] = os.path.join(self.test_env.log_dir, engine["id"] + ".log")
```

Comment:
This was tested in https://jenkins-3.daos.hpc.amslabs.hpecorp.net/job/daos-stack/job/daos/job/PR-16764/3/artifact/Functional%20Hardware%20Large%20MD%20on%20SSD/control/config_generate_run.py/job.log and ran into this issue:
```
2025-08-27 07:55:15,683 stacktrace       L0042 ERROR| Reproduced traceback from: /localhome/jenkins/venv/lib64/python3.6/site-packages/avocado/core/test.py:767
2025-08-27 07:55:15,683 stacktrace       L0045 ERROR| Traceback (most recent call last):
2025-08-27 07:55:15,683 stacktrace       L0045 ERROR|   File "/usr/lib/daos/TESTING/ftest/control/config_generate_run.py", line 71, in test_config_generate_run
2025-08-27 07:55:15,683 stacktrace       L0045 ERROR|     engine["log_file"] = os.path.join(self.test_env.log_dir, engine["id"] + ".log")
2025-08-27 07:55:15,683 stacktrace       L0045 ERROR| KeyError: 'id'
```


I'd suggest this fiix:
```suggestion
            engine["log_file"] = os.path.join(self.test_env.log_dir, os.path.basename(engine["log_file"]))
```